src/model/id.rs

```diff
@@ -339,7 +339,7 @@ id_u64! {
 /// This identifier is special, it simply models internal IDs for type safety,
 /// and therefore cannot be [`Serialize`]d or [`Deserialize`]d.
```

Comment:
If you're gonna do it, this comment is outdated?

Reply:
Yep, this comment should be `cannot be deserialized.`

Reply:
Fixed